### ISSUE TITLE
Jupyter notebook examples

### ISSUE THREAD

**Issue by StevE-Ong** [berceanu/prepic#173]
I think it would be good to have a Jupyter notebook examples

**Comment by berceanu** [comment]
I agree. This would be a good opportunity for @VadimBim, as he is learning to use prepic.

**Comment by berceanu** [comment]
A good starting point are the [examples](https://github.com/berceanu/prepic/tree/a6050a9c7b7cc5ea64af2a67db8d7a03ca448d25/examples)
